Sources/NIO/UniversalBootstrapSupport.swift

```diff
@@ -158,6 +165,11 @@ public struct NIOClientTCPBootstrap {
         self.underlyingBootstrap = bootstrap
         self.tlsEnablerTypeErased = tlsEnabler
     }
+    
+    internal init(_ original : NIOClientTCPBootstrap, withUpdated underlying : NIOClientTCPBootstrapProtocol) {
```

Comment:
[just in case you're making more changes anyway], `with` is typically not used for those argument labels. I guess more standard would be `updating underlying: ...`